### ISSUE TITLE
Slicer causes printer bootloop firmware_E400_CREALITY_422_BLTOUCH.bin

### ISSUE THREAD

**Issue by Jonathanblanke** [enderextender/FirmwareReleases#11]
firmware_E400_CREALITY_422_BLTOUCH.bin. I have many problems with this most I can work around , but the deal breaker is that every time I load up Crealty slicer or Cura the machine goes into a complete bootloop  and is not seen by either slicer . the drivers are installed correctly in windows 11 .. when I flash back to OEM FW the problem goes away . at the moment I would rather have a PC connection over anything else.  . The printer is an Ender 3 neo

**Comment by enderextender** [comment]
This means your y axis is not positioned right. I will look into the booting issue with the firmware.

Regards,
Mark Williamson
________________________________
From: John Blanke ***@***.***>
Sent: Monday, April 24, 2023 9:51:12 PM
To: enderextender/FirmwareReleases ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [enderextender/FirmwareReleases] Slicer causes printer bootloop firmware_E400_CREALITY_422_BLTOUCH.bin (Issue #11)


firmware_E400_CREALITY_422_BLTOUCH.bin. I have many problems with this most I can work around , but the deal breaker is that every time I load up Crealty slicer or Cura the machine goes into a complete bootloop and is not seen by either slicer . the drivers are installed correctly in windows 11 .. when I flash back to OEM FW the problem goes away . at the moment I would rather have a PC connection over anything else. The other issues are with the stopper locations . I had to relocate the switches or else the bed would move too far back and the bltouch would not be over the build plate making it imposable to auto level

**Comment by Jonathanblanke** [comment]
I thank you for your help, Currently I am using this FW but I do miss the PC connection

**Comment by Jonathanblanke** [comment]
also what ever I try the printer will not print centered
Maybe there is a setting I have missed but I have all the probe offsets set and the auto leveling works but even the purge line at the start of the print is about 10 mm off the side of the board

**Comment by Jonathanblanke** [comment]
@enderextender 
setting my x.y off sets corrects the auto leveling problem . but the only way I have found to center the prints and even print the purge line on the bed is to stack 3 "5mm" magnets on the left side of the hot end housing so it hits the switch sooner . I don't understand why auto leveling works fine but when it comes to actually printing everything is shifted 15+mm to the left. even with the 15mm shims the purge line is barely on the bed.
![Screenshot 2023-04-28 022833](https://user-images.githubusercontent.com/87837345/235083215-be44297f-03f7-44e6-a974-2801cac526cb.png)

Edit , Also the bootloop when opening any slicer while connected to pc still exists

**Comment by enderextender** [comment]
Have you set the probe offsets correctly in the firmware and your slicer?  Did you clear firmware (reset) in the lcd after updating firmware?


Regards,
Mark Williamson
Ender Extender Support

**Comment by Jonathanblanke** [comment]
yes the probe offsets are set and I did clear the firmware after flashing . Like I said, with the probe offsets set the auto leveling works but the prints are all off center to the left .. and still boot loops when any slicer is loaded . although the x offset is at -70 to keep it over the build plate which seems very abnormal to me because it is way off the measurements plus if I take the 15mm magnet off and just set it -15 more its still not on the bed

**Comment by Jonathanblanke** [comment]
should I be using https://github.com/enderextender/FirmwareReleases/blob/main/Creality-4.2.2/firmware_E300_CREALITY_422_BLTOUCH_ZMIN.bin ? I figured no because I have 5 wires on the BLtouch .. but I'm new to all this and could be mistaken

**Comment by Jonathanblanke** [comment]
and now after each print the z offset resets 10mm up and I have to baby step it back down to the bed
![WIN_20230501_04_34_43_Pro](https://user-images.githubusercontent.com/87837345/235435865-0dd65bc6-2fdf-4b86-98dc-9b5512269ba7.jpg)

**Comment by Jonathanblanke** [comment]
This is driving me crazy . I need the PC connection to work 

https://user-images.githubusercontent.com/87837345/235547726-566f3d96-5a16-4f62-9d29-95a9c7dba425.mp4

this was an expensive upgrade, The Firmware should not be this buggy

**Comment by Jonathanblanke** [comment]
@enderextender any luck with the bootloop problem?

**Comment by Jonathanblanke** [comment]
ok now its offset to the right just as much as it was to the left and will try to print off the build plate but it still drops the purge line off of the left side of the build plate . I have the dimensions in creality slicer at 394x394 .
